src/internal/m365/collection/drive/delta_tree.go

```diff
@@ -317,8 +313,7 @@ func (face *folderyMcFolderFace) setPreviousPath(
 // this func will update and/or clean up all the old references.
 func (face *folderyMcFolderFace) addFile(
 	parentID, id string,
```

Comment:
not sure if we want to do this, but it seems like we could just replace all the args with `file` and source the parentID and itemID from that too